M/mimalloc/build_tarballs.jl

```diff
@@ -3,12 +3,12 @@
 using BinaryBuilder, Pkg
 
 name = "mimalloc"
-version = v"2.0.1"
+version = v"2.0.6"
 
 # Collection of sources required to complete build
 sources = [
     GitSource("https://github.com/microsoft/mimalloc.git",
-              "8e35ccc43be293a9bfd6e63da310a79c235d25d9"),
+              "82712f4a8f038a7fb4df2790f4c3b7e3ed9e219b"),
```

Comment:
Where did you get that?  https://github.com/microsoft/mimalloc/commit/82712f4a8f038a7fb4df2790f4c3b7e3ed9e219b

Reply:
I presume you want https://github.com/microsoft/mimalloc/commit/f2712f4a8f038a7fb4df2790f4c3b7e3ed9e219b?

Reply:
That's a copy pasting failure :|